scala/build.sbt

```diff
@@ -10,7 +10,7 @@ import scala.scalanative.build._
 
 // defaults set with common options shown
 nativeConfig ~= { c =>
-  c.withLTO(LTO.full) // thin
+  c.withLTO(LTO.none) // thin
```

Comment:
We can't just skip these lines to leave things at defaults?

Reply:
Well, the project was generated using `sbt new scala-native/scala-native.g8` as per docs (https://scala-native.org/en/stable/user/sbt.html)
I didn't add these lines myself, and it seems to work fine without them but we at least need "Release" mode
I.e.
```
// defaults set with common options shown
nativeConfig ~= { c =>
  c.withMode(Mode.release)
}```

Reply:
I made the wrong call on this. If this file was generated with `sbt new scala-native/scala-native.g8`, these settings are the language defaults for native compilation (i.e. it was decided that the expected behavior is to have slow builds, but better perf - a tradeoff others don't do, but that's fine). Sizegame should be measuring the experience of an average developer and average developer is not going to go and edit these unless there's a need. I'll make a PR to put back the original.